lua/core/clock.lua

```diff
@@ -51,6 +51,12 @@ clock.sync = function(...)
   return coroutine.yield(SCHEDULE_SYNC, ...)
 end
 
+--- static callback when clock tempo is adjusted via PARAMETERS > CLOCK > tempo;
+-- user scripts can redefine
+-- @static
+-- @param bpm : the new tempo
+-- @param source: the clock source
+function clock.tempo_change_handler(bpm,source) end
```

Comment:
maybe this definition could be moved together with the other callbacks (clock.transport) and declared as `clock.tempo_change_handler = function...` for consistency with the rest of the module?

also see the comment about resetting this function to `nil` below, it could be initialized as `nil` value too.